build.psm1

```diff
@@ -220,6 +220,7 @@ function Start-PSBuild {
         # If this parameter is not provided it will get determined automatically.
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Get-Service can get the 'UserName' of a service
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 304
304:             $service | Should -Not -BeNullOrEmpty
```